scripts/ci/pre_warm_models.py

```
@@ -7,42 +7,42 @@
+        print"Models pre-warmed successfully!"
         return True
 
     except Exception as e:
-        print(f"Error pre-warming models: {e}")
+        printf"Error pre-warming models: {e}"
         return False
 
 if __name__ == "__main__":
     success = pre_warm_models()
-    sys.exit(0 if success else 1) 
+    sys.exit0 if success else 1 
```

Ruff (0.12.2)

Trailing whitespace

(W291)